contracts/swap/swap.go

```diff
@@ -48,8 +49,10 @@ type Contract interface {
 	Withdraw(auth *bind.TransactOpts, amount *big.Int) (*types.Receipt, error)
 	// Deposit sends a raw transaction to the chequebook, triggering the fallback—depositing amount
 	Deposit(auth *bind.TransactOpts, amout *big.Int) (*types.Receipt, error)
-	// CashChequeBeneficiary cashes the cheque by the beneficiary
-	CashChequeBeneficiary(auth *bind.TransactOpts, beneficiary common.Address, cumulativePayout *big.Int, ownerSig []byte) (*CashChequeResult, *types.Receipt, error)
+	// CashChequeBeneficiaryStart sends the transaction to a cash a cheque as the beneficiary
```

Comment:
Small typo here: "to a cash a cheque"

Reply:
fixed

Reply:
fixed now. the same comment existed in two places and I only had fixed one.